src/frontend/jest.config.js

```diff
@@ -3,7 +3,11 @@ module.exports = {
   moduleNameMapper: {
     '\\.(css)$': '<rootDir>/__mocks__/styleMock.js',
   },
-  setupFilesAfterEnv: ['./testSetup.ts'],
```

Comment:
In your commit description: `s/We also need to many more polyfills/We also need to add many more polyfills/`